htdocs/install/mysql/migration/22.0.0-23.0.0.sql

```diff
@@ -31,20 +31,75 @@
+	rowid                     integer AUTO_INCREMENT PRIMARY KEY,
+	tms                       timestamp DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+	fk_object                 integer NOT NULL,
+	import_key                varchar(14)                                 -- import key
 ) ENGINE=innodb;
 
 ALTER TABLE llx_paiement_extrafields ADD UNIQUE INDEX uk_paiement_extrafields (fk_object);
 
+CREATE TABLE llx_c_accounting_analytic_axis (
+	rowid               integer         AUTO_INCREMENT PRIMARY KEY,
+	label               varchar(255)    NOT NULL,
+	code                varchar(32)     NOT NULL,
+	active              integer         DEFAULT 1,
+	entity              integer         DEFAULT 1 NOT NULL,
+	datec               datetime,
+	tms                 timestamp DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+	fk_user_author      integer         NOT NULL,
+	fk_user_modif       integer,
+	import_key          varchar(14)
+) ENGINE=innodb;
+
+ALTER TABLE llx_c_accounting_analytic_axis ADD UNIQUE INDEX uk_c_accounting_analytic_axis(code, entity);
+
+CREATE TABLE llx_c_accounting_analytic_account (
+	rowid               integer         AUTO_INCREMENT PRIMARY KEY,
+	label               varchar(255)    NOT NULL,
+	code                varchar(32)     NOT NULL,
+	description			text,
+	fk_axis				integer			NOT NULL,
+	active              integer         DEFAULT 1,
+	entity              integer         DEFAULT 1 NOT NULL,
+	date_start			date,
+	date_end			date,
+	datec               datetime,
+	tms                 timestamp DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+	fk_user_author      integer         NOT NULL,
+	fk_user_modif       integer,
+	import_key          varchar(14)
+) ENGINE=innodb;
+
+ALTER TABLE llx_c_accounting_analytic_account ADD UNIQUE INDEX uk_c_accounting_analytic_account(code, entity);
+ALTER TABLE llx_c_accounting_analytic_account ADD CONSTRAINT fk_accounting_analytic_account_fk_axis FOREIGN KEY (fk_axis) REFERENCES llx_c_accounting_analytic_axis (rowid);
+
+
+CREATE TABLE llx_accounting_analytic_distribution (
+	rowid				integer			AUTO_INCREMENT PRIMARY KEY,
+	fk_source_line		integer			NOT NULL,		-- id de la ligne source (facture, écriture, etc.)
+	sourcetype			varchar(32)		NOT NULL,		-- ex: 'facturedet', 'accounting_line'
+	fk_analytic_account integer			NOT NULL,
+	percentage			real			DEFAULT 100,	-- ex: 50.0 pour 50%
+	amount				double(24,8),					-- optional, if you prefer to work on the amount
+	entity				integer			DEFAULT 1,
+	datec               datetime,
+	tms                 timestamp DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+	fk_user_author      integer         NOT NULL,
+	fk_user_modif       integer,
+	import_key          varchar(14)
+) ENGINE=innodb;
+
+ALTER TABLE llx_accounting_analytic_distribution ADD CONSTRAINT fk_accounting_analytic_distribution_fk_analytic_account FOREIGN KEY (fk_analytic_account) REFERENCES llx_accounting_analytic_distribution (rowid);
```

Comment:
I think what you want to do is not
`ALTER TABLE llx_accounting_analytic_distribution ADD CONSTRAINT fk_accounting_analytic_distribution_fk_analytic_account FOREIGN KEY (fk_analytic_account) REFERENCES llx_accounting_analytic_distribution (rowid);
`
But
`ALTER TABLE llx_accounting_analytic_distribution ADD CONSTRAINT fk_accounting_analytic_distribution_fk_analytic_account FOREIGN KEY (fk_analytic_account) REFERENCES llx_c_accounting_analytic_account (rowid);
`
Can you confirm ?